courses/gnatcheck/010_gnatcheck.rst

```diff
@@ -85,53 +85,163 @@ Conformance To Standards Requirement - DO-178
   + Those units named in :ada:`with`-clauses, transitively
   + Whether or not they are to be analyzed themselves
 
+=============
+Basic Usage
+=============
+
 -------------------------
 Command Line Invocation
 -------------------------
 
-.. code::
+.. container:: latex_environment tiny
+
+.. container:: latex_environment scriptsize
 
-    gnatcheck [options]
-             <filename> | -files=<filenames_file>
-             -rules
-                rule_switches | -from=file
+  .. code::
 
-* :code:`<filename>`
+    gnatcheck [options] {filename} {-files=filename} [-cargs gcc_switches] -rules rule_switches
 
-   - Source file names; wildcards allowed
-   - Mutually exclusive with :code:`-files=<filenames_file>`
+.. list-table::
+   :header-rows: 1
 
-      + :code:`filenames_file` is a file containing the source files names
+   * - Argument
 
-* :code:`-rules`
+     - Description
 
-   - Specific rules switches
-   - Or name of a text file containing them: :code:`-from=<filename>`
+   * - {filename}
 
-.. container:: latex_environment footnotesize
+     - File to analyze (wildcards allowed)
+
+   * - {files=filename}
+
+     - :filename:`filename` specifies text file containing list of files to analyze
+
+   * - -rules rule_switches
+
+     - Rules to apply for analysis
+
+Where ``rule_switches`` can be any combination of the following:
+
+  .. list-table::
+     :header-rows: 1
+
+     * - Switch
+
+       - Explanation
+
+     * - -from=filename
 
-   :command:`gnatcheck main.adb  -rules +RPositional_Parameters`
-   :command:`gnatcheck -P foo.gpr`
+       - read rule options from :filename:`filename`
+
+     * - +R<rule_id>[:param]
+
+       - turn ON a given rule [with given parameter]
+
+     * - -R<rule_id>
+
+       - turn OFF a given rule
+
+     * - -R<rule_id>:param
+
+       - turn OFF some of the checks for a given  rule,
+
+     * - 
+
+       - depending on the specified parameter
 
 ----------------------
-Some Useful Switches
+Some Useful Options
 ----------------------
-* :code:`--help`, :code:`-h`
 
-  + List rule identifiers with very brief descriptions
+.. list-table::
+   :header-rows: 1
+
+   * - Option
+
+     - Description
+
+   * - --help
```

Comment:
Why remove the `:code:` role?

Reply:
Put it back (as literal, not code, but same presentation)